### ISSUE TITLE
An assembly specified in the application dependencies manifest (CMD.deps.json) was not found

### ISSUE THREAD

**Issue by ijmiller2** [smith-chem-wisc/FlashLFQ#81]
Running this command:

```bash
/home/ijmiller2/.dotnet/dotnet CMD.dll
```

I'm getting this error:

```
Error:
  An assembly specified in the application dependencies manifest (CMD.deps.json) was not found:
    package: 'FluentCommandLineParser', version: '1.5.0.20-commands'
    path: 'lib/net35/FluentCommandLineParser.dll'
```

In case it's relevant, this is with the latest release and here's what: ```/home/ijmiller2/.dotnet/dotnet --info``` produces:

```.NET Core SDK (reflecting any global.json):
 Version:   2.1.802
 Commit:    177d0b2525

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  18.04
 OS Platform: Linux
 RID:         ubuntu.18.04-x64
 Base Path:   /home/ijmiller2/.dotnet/sdk/2.1.802/

Host (useful for support):
  Version: 2.1.13
  Commit:  1a165a1588

.NET Core SDKs installed:
  2.1.802 [/home/ijmiller2/.dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.13 [/home/ijmiller2/.dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.13 [/home/ijmiller2/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.13 [/home/ijmiller2/.dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download```

**Comment by rmillikin** [comment]
Thanks for the crash report! I will check this out tomorrow. I'm curious if this is a Linux-only issue or if it also occurs on Windows. I'll investigate....

**Comment by ijmiller2** [comment]
Great, thank you!

**Comment by rmillikin** [comment]
I was able to reproduce this issue in Linux (Ubuntu). All versions of FlashLFQ (.NET Core command-line, .NET Framework command-line, and .NET Framework GUI) work in Windows, though. I will try to fix this Linux issue ASAP.

**Comment by rmillikin** [comment]
This issue was caused by Visual Studio not building .NET core apps properly; the dependent nuget packages were not packaged properly with the .NET core release. I've fixed that. I also switched command-line parsers to fix issues #72 and #71 . I'll make a new release within a few days after some more bugtesting.

**Comment by ijmiller2** [comment]
Excellent! Thanks for the quick turnaround. I look forward to testing it out.

**Comment by rmillikin** [comment]
This should be fixed with version 1.0.1, which I just released. Please give that a try and open a new issue if you encounter other issues. Thanks again for the report!
